share/test/data/messageV1/README.md

```diff
@@ -0,0 +1,45 @@
+BinaryMessage data in single object encoding https://avro.apache.org/docs/current/spec.html#single_object_encoding
+
+Ground truth data generated with Java Code
```

Comment:
There is no new code to run the Java test in CI.
See `.github/workflows/test-lang-rust-ci.yml`

Reply:
Do you think we should generate this data every time? I was looking at the `share/test/data/` folder for avro container files data and I wanted to create an extension to that for message format where the bytes are "stuck in time". So even if generated by Java at first they can be used to keep checking Rust against Java?

Reply:
Agreed! Let's keep it simple!

Reply:
OK, the `.bin` file will be pre-generated!
But we should run the new example as part of the interop tests, right ?

Reply:
done!